axelrod/fingerprint.py

```diff
@@ -65,168 +28,91 @@ def __init__(self, strategy, probe=axl.TitForTat):
             A class that must be descended from axelrod.Player or an instance of
             axelrod.Player.
             Default: Tit For Tat
+        step: float = 0.01
+            The separation between each Point. Smaller steps will
+            produce more Points that will be closer together.
+        progress_bar: bool = True
+            True = show a progress bar when constructing probes and running
+            tournament.
         """
-        self.strategy = strategy
-        self.probe = probe
+        self.player = create_player(strategy)
+        self._probe_class, self._probe_kwargs = get_class_and_kwargs(probe)
+        self._step = step
+        self.progress_bar = progress_bar
+        self.points = self._create_points()
+        self._probe_list = self._create_probes()
```

Comment:
on more thought, either way of thinking about it, i absolutely agree that probes creation belongs in fingerprint and, frankly, will be easy enough to move.  i have 1/2 baked thoughts on the matter, but first, some light reading :), #759